x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -97,6 +97,7 @@ export const uiSettings: Record<string, UiSettingsParams<boolean | number | stri
   },
   [defaultApmServiceEnvironment]: {
     category: [observabilityFeatureId],
+    sensitive: true,
```

Comment:
Is this really a sensitive setting? @elastic/apm-ui

Reply:
It can be considered PII, for example when we store telemetry data from KQL bar, we strip the value from all fields including service environment.

Reply:
Thank you for confirming! I'll keep it as `sensitive: true` then, so we redact any custom values users set in this UI Setting 😇